src/sigmatcher/definitions.py

```diff
@@ -2,8 +2,11 @@
 import re
 import sys
 from abc import ABC, abstractmethod
+from functools import cached_property
 from pathlib import Path
-from typing import List, Optional, Tuple, Union
+from typing import ClassVar, Dict, List, Optional, Set, Tuple, Union
+
+from sigmatcher.results import Result
 
 if sys.version_info < (3, 9):
```

Comment:
graphlib is >=3.9
no need to handle older versions now

Reply:
(add `requires-python` to the pyproject.toml)

Reply:
For 3.8 I use a backport of graphlib (`backports-graphlib`)